### ISSUE TITLE
slowRead problems when submitting files larger then 30KB

### ISSUE THREAD

**Issue by goncalvesjoao** [h2non/toxy#42]
Hi guys, first of all thank you for building this cool tool!

I've been making a small, simple POC for myself using toxy to simulate bad network behaviour and I stumble upon a problem.

Using the poison slowRead like so:
```javascript
proxy
    .post('/upload')
    .poison(poisons.slowRead({ chunk: 102400, threshold: 500 }))
```

The browser becomes unable to submit files larger then 30KB, 70KB files sometimes get submitted 100KB for instance, doesn't.

You can see for yourselves: https://github.com/goncalvesjoao/up-your-ass-ets
the POC is fairly simple and quick to look at, am I missing something or am I configuring toxy in a wrong way?

Thanks,
John

**Comment by h2non** [comment]
Fixed in `0.3.10`. Try updating `toxy`.

**Comment by goncalvesjoao** [comment]
Thanks @h2non I just confirm it, everything is working! :metal:
